modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -248,6 +248,13 @@ if(CONFIG_NFCT_PINS_AS_GPIOS OR CONFIG_TFM_NFCT_PINS_AS_GPIOS)
     APPEND PROPERTY TFM_CMAKE_OPTIONS
     -DCONFIG_NFCT_PINS_AS_GPIOS=ON
   )
+  # HAL only checks for NRF_CONFIG_NFCT_PINS_AS_GPIOS on 54L
+  if(CONFIG_SOC_SERIES_NRF54LX)
```

Comment:
That looks ok, I am just wondering if the CONFIG_NFCT_PINS_AS_GPIOS is at all useful for nrf54l devices. If it is not I suggest to not add it to the 54l targets at all. Maybe just include in an else statement of this if?